things/postgres/init.go

```diff
@@ -15,11 +15,23 @@ import (
 	migrate "github.com/rubenv/sql-migrate"
 )
 
+type Config struct {
```

Comment:
Add comment to explain meaning of this config struct.

Reply:
Added a comment to both this one and users/postgres/init.go